packages/cosmos_ui_components/lib/components/cosmos_switch.dart

```diff
@@ -1,38 +1,63 @@
-import 'package:cosmos_ui_components/cosmos_theme.dart';
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:cosmos_ui_components/utils/global_constants.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 class CosmosSwitch extends StatelessWidget {
   const CosmosSwitch({
     required this.checked,
+    this.toggleAsset = 'assets/images/toggle.png',
+    this.checkedGradientStops = const [0, 0.5866, 1],
+    this.gradientColors = GlobalConstants.cosmosGradientColors,
+    this.package,
     this.onChanged,
     Key? key,
   }) : super(key: key);
 
   final bool checked;
   final ValueChanged<bool>? onChanged;
+  final String toggleAsset;
+  final List<double> checkedGradientStops;
+  final List<Color> gradientColors;
+  final String? package;
```

Comment:
can you confirm this is working in starport template or emeris without setting up `package`? otherwise we need to update it to:
```suggestion
  final String? package = packageName;
```

Reply:
There is a little bit of an issue here.

If we assign `packageName` as a default value, then whatever the consumer wants to add an asset of their own, they will have to pass null inside `packageName` parameter specifically, otherwise `CosmosSwitch` will start finding the consumer side asset inside its own assets due to defaulted `packageName`.

But I think in our case this is the only thing we can do.

Reply:
so maybe we could default toggleAsset to null, and then inside the widget do the following:

```dart
Image.asset(
   imageAsset ?? 'assets/images/toggle.png',
   package: package ?? (imageAsset == null ? packageName : null),
 )
 ```

Reply:
This would work I guess. Thanks